ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot

```diff
@@ -25,7 +26,7 @@ Verify RHODS Admins Can Import A Custom Serving Runtime Template By Uploading A
     Open Dashboard Settings    settings_page=Serving runtimes
     Upload Serving Runtime Template    runtime_filepath=${OVMS_RUNTIME_FILEPATH}
     ...    serving_platform=multi      runtime_protocol=gRPC
-    Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}
+    Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_XPATH_NAME}
```

Comment:
Where is the `displayed_name=${UPLOADED_OVMS_XPATH_NAME` varible name i f9 not see in the suite

Reply:
fixed it